libs/sdk-ui-filters/src/MeasureValueFilter/Dropdown.tsx

```diff
@@ -20,12 +21,13 @@ interface IDropdownOwnProps {
     operator?: MeasureValueFilterOperator;
     value?: IMeasureValueFilterValue;
     usePercentage?: boolean;
-    warningMessage?: string;
+    warningMessage?: WarningMessage;
     locale?: string;
     anchorEl: EventTarget | string;
     separators?: ISeparators;
     displayTreatNullAsZeroOption?: boolean;
     treatNullAsZeroValue?: boolean;
+    isOperatorDisabled?: boolean;
```

Comment:
Maybe switch it around? `enableOperatorSelection` with default `true`?
cc  @yanlipnican

Reply:
I think that operator is usually enabled and people won't disable it very often. So isOperatorDisabled is better.

Reply:
but enableOperatorSelection with default true is also great 😄  ... I would say make it consistent and there is already displayTreatNullAsZeroOption and usePercentage  and treatNullAsZeroValue so